docs/UserGuide.md

```diff
@@ -10,12 +10,13 @@ title: User Guide
 3. [Features](#3-features)<br>
     3.1 [Command Format](#31-command-format)<br>
     3.2 [Add a patient: `addpatient`](#32-add-a-patient-addpatient)<br>
-    3.3 [Edit a patient: `editpatient`](#33-edit-a-patient-editpatient)<br>
-    3.4 [Search information: `searchpatient`](#34-search-information-searchpatient)<br>
-    3.5 [Add rooms in hotel: `addRooms`](#35-add-rooms-in-hotel-addrooms)<br>
-    3.6 [Find the first free room: `findRoom`](#36-find-the-first-free-room-findroom)<br>
-    3.7 [View help: `help`](#37-view-help-help)<br>
-    3.8 [Save the data](#38-save-the-data)<br>
+    3.3 [Delete a patient: `deletepatient`](#33-delete-a-patient-deletepatient)<br>
```

Comment:
I like how there is an order for the different types of entities (patients followed by rooms followed by miscellaneous). Maybe in the future, we can a header to each entity to further organize the content of our user guide? E.g.

```
3.1 [Command Format](#31-command-format)<br>
3.2 [Patients](<Section explaining what Patients are>)<br>
     3.2.1 [Add a patient: `addpatient`](#32-add-a-patient-addpatient)<br>
// ...
```

Reply:
Yes, good suggestion. We definitely should adhere to this in the next UG update!